examples/form/headless/saveAndContinue/src/pages/posts/edit.tsx

```diff
@@ -0,0 +1,162 @@
+import { useEffect, useState } from "react";
```

Comment:
```suggestion
import React, { useEffect, useState } from "react";
```

same changes as the ones in the docs.